release-notes/20221103-privileged-scc.md

```diff
@@ -0,0 +1,3 @@
+The privileged SCC permission for Submariner components in OCP is set now by 
+creating a separate ClusterRole and ClusterRoleBinding resources instead of 
```

Comment:
Trim the trailing spaces:

```suggestion
The privileged SCC permission for Submariner components in OCP is set now by
creating a separate ClusterRole and ClusterRoleBinding resources instead of
```